Godeps/_workspace/src/k8s.io/kubernetes/pkg/client/unversioned/clientcmd/loader.go

```diff
@@ -238,7 +237,6 @@ func LoadFromFile(filename string) (*clientcmdapi.Config, error) {
 	if err != nil {
 		return nil, err
 	}
-	glog.V(3).Infoln("Config loaded from file", filename)
```

Comment:
@deads2k @liggitt the extended tests logs are flooded by these messaging. It will only gives you what kubeconfig is being used. In case of mysql or other extended tests, this will print out >100 messages to log ;-)